cmd/litcli/autopilot.go

```diff
@@ -51,25 +54,36 @@ var addAutopilotSessionCmd = cli.Command{
 			Name:     "feature",
 			Required: true,
 		},
-		cli.StringFlag{
-			Name: "channel-restrict-list",
-			Usage: "list of channel IDs that the " +
-				"Autopilot server should not " +
-				"perform actions on. In the " +
-				"form of: chanID1,chanID2,...",
+		cli.StringSliceFlag{
+			Name: "feature-channel-restrict-list",
+			Usage: `List of channel IDs that the ` +
+				`Autopilot server should not ` +
+				`perform actions on. In the ` +
+				`form of: [chanID1,chanID2,...]. ` +
```

Comment:
nit: 
Should be as below, correct?

```suggestion
				`form of: ["chanID1","chanID2",...]. ` +
```

Reply:
ah that was intentional, we would error to parse into []int64 otherwise

Reply:
while we are here, let's clear up this confusion by specifying that this should be a list of numbers. Since even in LND, there is a `ChannelID` which is a uint64 and another which is a string ([32]byte)

Reply:
a disadvantage of the generic flags is that it's not as well documented